README.md

```diff
@@ -1,5 +1,7 @@
 # Shark Cage Team project
 
+[![Build status](https://ci.appveyor.com/api/projects/status/xinpyru7ij60bja3?svg=true)](https://ci.appveyor.com/project/SailReal/htwg-shark-cage)
```

Comment:
Can the appveyor URL be changed to use the organization Name instead of your personal account?

Reply:
Have to create an new account for that but you're right, will do that 👍